Sources/StevensonCore/GHRepo.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import Vapor
+
+typealias Version = String
```

Comment:
Let's maybe use `Tagged`? 😄

Reply:
Do you mean add it as dependency, or just declare a `struct Tagged<T> { let id: String }` locally and use it?